telemetry/packages/server/src/modules/public-data/PublicData.service.ts

```diff
@@ -0,0 +1,119 @@
+import { HttpException, Injectable, LoggerService } from '@nestjs/common';
+import { Logger } from '../logger/Logger.decorator';
+import { INFLUX_TELEMETRY_BUCKET } from '../core/config';
+import { flux } from '@influxdata/influxdb-client';
+import { InfluxService } from '../influx/Influx.service';
+import { ACTIVE_STATES } from '@hyped/telemetry-constants';
+
+type InfluxRow = {
+  _time: string;
```

Comment:
Do you mind creating a common type `InfluxRow`, shoving it somewhere and then extend from it here? I can see a few other `InfluxRow`s around the codebase

Reply:
Will do 👍